internal/service/alertconfiguration/data_source_alert_configuration_test.go

```diff
@@ -41,23 +36,19 @@ func TestAccConfigDSAlertConfiguration_basic(t *testing.T) {
 
 func TestAccConfigDSAlertConfiguration_withThreshold(t *testing.T) {
 	var (
-		alert          = &admin.GroupAlertsConfig{}
-		dataSourceName = "data.mongodbatlas_alert_configuration.test"
-		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName    = acc.RandomProjectName()
+		projectID = acc.ProjectIDExecution(t)
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:                 func() { acc.PreCheckBasic(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: configWithThreshold(orgID, projectName, true, 1),
+				Config: configWithThreshold(projectID, true, 1),
```

Comment:
[q] taking this test as an example the terraform configuration is creating a single alert configuration and running a data source for that specific alert, could we preserve running this in parallel?
I would asume the only test cases that needs to be run sequentially are those that rely on asserting the plural data source, and effectively check a certain amount of resources. This also applies for database user tests that where adjusted from ParallelTest to Test

Reply:
+1

Reply:
@AgustinBettati great callout!!! i've changed and it worked